### PR TITLE
LPS-153893 Read typeOptions.displayType property

### PR DESCRIPTION
**Pull request by p2kmgcl**

### CODE REVIEW

**Comment by liferay-echo** [comment]
Thank you for submitting this PR. 👏

Make sure it follows the [Echo Pull Request Template](https://gist.github.com/liferay-echo/5a4455285ff1800f9da146816d85e67d#file-pull-request-template-md) if you haven't already done so. 

Learn more about it [here](https://liferay.atlassian.net/wiki/spaces/ENGECHO/pages/2027455014/Echo+Pull+Request+Etiquette).

**Comment by p2kmgcl** [comment]
ci:test:sf

**Comment by p2kmgcl** [comment]
ci:test:relevant

**Comment by victorg1991** [comment]
ci:forward

File: modules/apps/fragment/fragment-service/src/main/resources/com/liferay/fragment/internal/validator/dependencies/configuration-json-schema.json [hunk 1]
```diff
@@ -175,6 +175,21 @@
 						"boolean",
 						"string"
 					]
+				},
+				"typeOptions": {
+					"additionalProperties": false,
+					"properties": {
+						"displayType": {
```
Comment (victorg1991) [inline modules/apps/fragment/fragment-service/src/main/resources/com/liferay/fragment/internal/validator/dependencies/configuration-json-schema.json:182]: I think we may need to document this

**Comment by ealonso** [comment]
ci:forward

**Comment by ealonso** [comment]
ci:forward

**Comment by ealonso** [comment]
ci:forward

**Pull request closed without merging**